diffsims/utils/sim_utils.py

```diff
@@ -448,3 +448,294 @@ def uvtw_to_uvw(uvtw):
+    --------
+    >>> import numpy as np
+    >>> import diffsims.utils.sim_utils as sim_utils
+    >>> data = np.random.random((100, 100))  # In radians
+    >>> acceleration_voltage = 200000  # 200 kV (in Volt)
+    >>> bst = sim_utils.beta_to_bst(data, 200000)
+
+    """
+    wavelength = acceleration_voltage_to_wavelength(acceleration_voltage)
+    beta = beam_deflection
+
+    Bst = beta * h / (wavelength * e)
+    return Bst
+
+
+def tesla_to_am(data):
+    """Convert data from Tesla to A/m
+
+    Parameters
+    ----------
+    data : NumPy array
+        Data in Tesla
+
+    Returns
+    -------
+    output_data : NumPy array
+        In A/m
+
+    Examples
+    --------
+    >>> import numpy as np
+    >>> import diffsims.utils.sim_utils as sim_utils
+    >>> data_T = np.random.random((100, 100))  # In tesla
+    >>> data_am = sim_utils.tesla_to_am(data_T)
+
+    """
+    return data / sc.mu_0
+
+
+def acceleration_voltage_to_velocity(acceleration_voltage):
+    """Get relativistic velocity of electron from acceleration voltage.
+
+    Parameters
+    ----------
+    acceleration_voltage : float
+        In Volt
+
+    Returns
+    -------
+    v : float
+        In m/s
+
+    Example
+    -------
+    >>> import diffsims.utils.sim_utils as sim_utils
+    >>> v = sim_utils.acceleration_voltage_to_velocity(200000) # 200 kV
+    >>> round(v)
+    208450035
+
+    """
+    av = acceleration_voltage
+    ##removed defenitions of constants as defined at the top
+    part1 = (1 + (av * e) / (m_e * c ** 2)) ** 2
+    v = c * (1 - (1 / part1)) ** 0.5
+    return v
+
+
+def acceleration_voltage_to_relativistic_mass(acceleration_voltage):
+    """Get relativistic mass of electron as function of acceleration voltage.
+
+    Parameters
+    ----------
+    acceleration_voltage : float
+        In Volt
+
+    Returns
+    -------
+    mr : float
+        Relativistic electron mass
+
+    Example
+    -------
+    >>> import diffsims.utils.sim_utils as sim_utils
+    >>> mr = sim_utils.acceleration_voltage_to_relativistic_mass(200000) # 200 kV
+
+    """
+    av = acceleration_voltage
+    ##removed defenitions of constants as defined at the top
+    v = acceleration_voltage_to_velocity(av)
+    part1 = 1 - (v ** 2) / (c ** 2)
+    mr = m_e / (part1) ** 0.5
+    return mr
+
+
+def et_to_beta(et, acceleration_voltage):
+    """Calculate beam deflection (beta) values from E * t.
+
+    Parameters
+    ----------
+    et : NumPy array
+        Electric field times thickness t of the sample.
+    acceleration_voltage : float
+        In Volts
+
+    Returns
+    -------
+    beta: NumPy array
+        Beam deflection in radians
+
+    Examples
+    --------
+    >>> import numpy as np
+    >>> import diffsims.utils.sim_utils as sim_utils
+    >>> data = np.random.random((100, 100))
+    >>> acceleration_voltage = 200000  # 200 kV (in Volt)
+    >>> beta = sim_utils.et_to_beta(data, acceleration_voltage)
+
+    """
+    av = acceleration_voltage
```

Comment:
this is generally a needless style, you can just put `acceleration_voltage` into the next two function